tests/integration/test_types_integration.py

```diff
@@ -156,6 +156,8 @@ def test_array(trino_connection):
     SqlTest(trino_connection) \
         .add_field(sql="CAST(null AS ARRAY(VARCHAR))", python=None) \
         .add_field(sql="ARRAY['a', 'b', null]", python=['a', 'b', None]) \
+        .add_field(sql="ARRAY[1.2, 2.4, null]", python=[Decimal("1.2"), Decimal("2.4"), None]) \
```

Comment:
"Refactor value mappers to separate classes"

"Refactoring" commit should focus on refactoring. It would be better to change the commit title.

Reply:
Done